### PR TITLE
CU-24pb6x1 - Change expo implementation

### PR DESCRIPTION
**Pull request by stepniaczky**

### CODE REVIEW

**Comment by pmakowski23** [comment]
Task linked: [CU-24pb6x1 Change expo implementation](https://app.clickup.com/t/24pb6x1)

File: components/sponsors/sponsors-grid.tsx [hunk 1]
```diff
@@ -6,8 +6,9 @@ import styles from './sponsors-grid.module.css';
 
 function SponsorCard({ sponsor }: { sponsor: Sponsor }) {
   return (
-    <Link key={sponsor.name} href={`/expo/${sponsor.slug}`}>
+    <Link key={sponsor.name} href={`${sponsor.website}`}>
```
Comment (pmakowski23) [inline components/sponsors/sponsors-grid.tsx:9]: You can just use `{sponsor.webiste}`. Template literal is unnecessary.

**Review by pmakowski23 — COMMENTED** [review]
Great work! Just one small comment ;)

File: components/sponsors/sponsors-grid.tsx (hunk 1 above)
Comment (stepniaczky) [reply components/sponsors/sponsors-grid.tsx:9]: Right

**Review by pmakowski23 — APPROVED** [review]

**Review by mdzieciuchowicz — APPROVED** [review]

**Pull request merged**